src/routers/features/featureFlagUtils.ts

```diff
@@ -0,0 +1,74 @@
+import { IncomingHttpHeaders } from "http";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import * as A from "fp-ts/lib/Array";
+import * as E from "fp-ts/lib/Either";
+import { PatternString } from "@pagopa/ts-commons/lib/strings";
+import { backendStatus } from "../../payloads/backend";
+import { VersionPerPlatform } from "../../../generated/definitions/content/VersionPerPlatform";
+import { BackendStatus } from "../../../generated/definitions/content/BackendStatus";
+
+type DeviceOS = {
+  iPhone: keyof VersionPerPlatform;
+  Android: keyof VersionPerPlatform;
+};
+
+const osPerDevice: DeviceOS = {
+  iPhone: "ios",
+  Android: "android"
+};
+
+type FeatureFlagWithMinAppVersion = Extract<
+  keyof BackendStatus["config"],
+  {
+    [K in keyof BackendStatus["config"]]: BackendStatus["config"][K] extends
+      | { min_app_version?: VersionPerPlatform }
+      | undefined
+      ? K
+      : never;
+  }[keyof BackendStatus["config"]]
+>;
```

Comment:
```suggestion
type FeatureFlagWithMinAppVersion<T> = Extract<
  keyof T,
  {
    [K in keyof T]: T[K] extends
      | { min_app_version?: VersionPerPlatform }
      | undefined
      ? K
      : never;
  }[keyof T]
>;
```

We could make it generic, what do you think?